### PR TITLE
Refresh of the how-tos on AWS IAM and metrics server.

### LINKED ISSUE #346: Review and update HT/Ops/Metrics

### PR DESCRIPTION
**Pull request by jhkrug**
Will close #346 and #347 when merged.

### CODE REVIEW

File: docs/howtos/operations/aws_iam_integration.md [hunk 1]
```diff
@@ -43,57 +49,66 @@ aws iam create-policy \
...
 
-## Authentication with AWS EKS Service Account
+## Authentication with an AWS EKS service account
 
-For increased security the policy can be attached to just the Epinio pod ServiceAccount. You will also need to create a specific ServiceAccount that will be bound to the staging job with the `server.stagingServiceAccountName` value.
+EKS is the Elastic Kubernetes Service.
 
-To use AWS IAM roles for service accounts, an IAM OIDC provider must exist for your cluster's OIDC issuer URL.  
+For increased security you can attach the policy to the Epinio pod ServiceAccount.
+You also need to create a specific ServiceAccount that to bind to the staging job with the `server.stagingServiceAccountName` value.
```
Comment (andreas-kupries) [inline docs/howtos/operations/aws_iam_integration.md:88]:
```suggestion
You also need to create a specific ServiceAccount that you bind to the staging job with the `server.stagingServiceAccountName` value.
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]

File: docs/howtos/operations/install_metrics_server.md [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
 
-> kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+```console
+kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+```
+
+<!--TODO:
+Some more information please. Why is this for simple environments and what is
+meant by simple environments. What would be a complex environment and how might
+you go about installing a metrics server in one. Should this note be here?
```
Comment (andreas-kupries) [inline docs/howtos/operations/install_metrics_server.md:34]: Huh. I wrote that part of the docs ... looking at the referenced https://github.com/kubernetes-sigs/metrics-server I suspect I meant simple as `not a high availability setup`, `not a large cluster needing custom scaling setup`.

File: docs/howtos/operations/aws_iam_integration.md (hunk 1 above)
Comment (jhkrug) [reply docs/howtos/operations/aws_iam_integration.md:88]: Thanks, fixed!

File: docs/howtos/operations/install_metrics_server.md (hunk 2 above)
Comment (jhkrug) [reply docs/howtos/operations/install_metrics_server.md]: Thanks

**Pull request merged**